.swiftpm/xcode/xcshareddata/xcschemes/Amplify-Package.xcscheme

```diff
@@ -314,6 +314,20 @@
                ReferencedContainer = "container:">
             </BuildableReference>
          </BuildActionEntry>
+         <BuildActionEntry
```

Comment:
These changes are temporary added to aid in testing and generating small PRs. Will be removed before merging to main dev-preview branch